### PR TITLE
New background

### PR DESCRIPTION
**Pull request by ClJarvis**
Updated color scheme

### CODE REVIEW

File: css/buttons.css [hunk 1]
```diff
@@ -2,46 +2,59 @@
...
 }
 
 .btn-warning {
-  background-color: rgba(255, 204, 0, 0.99);
+  background-color: #2907EB;
 }
 
 .btn-warning:hover {
-  background-color: rgba(247, 203, 0, 0.86);
+  background-color: #08a6ff; /*rgba(247, 203, 0, 0.86); */
```
Comment (erinpagemd) [inline css/buttons.css]: let's get rid of these commented things.

File: css/buttons.css [hunk 2]
```diff
@@ -2,46 +2,59 @@
...
   border-color: black;
   border-width: 6px;
 }
 
 .btn-danger {
-  background-color: #b91200;
+  background-color: #3D5AFE; /* #b91200;*/
```
Comment (erinpagemd) [inline css/buttons.css]: same. unless you need to save it for some reason?

File: css/buttons.css [hunk 3]
```diff
@@ -2,46 +2,59 @@
...
 }
 
 .btn-danger:hover {
-  background-color: #982e3b;
+  background-color: #0B3678; /*#982e3b;*/
```
Comment (erinpagemd) [inline css/buttons.css]: same. remove commented code.

**Comment by erinpagemd** [comment]
remove the comments, and then i will show you how to squash!

**Comment by ClJarvis** [comment]
Removed comments

**Comment by erinpagemd** [comment]
:shipit:

**Pull request merged**